libraries/AP_GPS/AP_GPS.cpp

```diff
@@ -1105,6 +1105,10 @@ bool AP_GPS::get_lag(uint8_t instance, float &lag_sec) const
     // always enusre a lag is provided
     lag_sec = GPS_WORST_LAG_SEC;
 
+    if (instance > GPS_BLENDED_INSTANCE) {
```

Comment:
Nitpicking: Why not `>= GPS_MAX_INSTANCES` slightly clearer that the instance is out of bounds, and allows the blend instance to be moved at some point if we wanted to.

Reply:
The blended instance isn't counted in `GPS_MAX_INSTANCES` so it would have to be `> GPS_MAX_INSTANCES` - and that's the reason why I chose to use `GPS_BLENDED_INSTANCE` instead, it makes it clear in which methods you can use the "physical" instances vs use the blended too.

I'm perfectly fine changing it, but it's not going to be future proof as you wanted - in fact, having it use `GPS_BLENDED_INSTANCE` will make it more noticeable in case of the blended instance being moved.

Reply:
Nevermind my comment, I was mixing `GPS_MAX_INSTANCES` with `GPS_MAX_RECEIVERS`.

Fixed.